### ISSUE TITLE
Interaction between SetList:EARLY_RETURN and SetList::PHP_73

### ISSUE THREAD

**Issue by HenkPoley** [rectorphp/rector#6812]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | eaf1f250031db227ef2c623c16753a3642bfa2bb             |

## Minimal PHP Code Causing Issue

https://getrector.org/demo/1ec48731-b208-6464-9cc8-3dd3d155fb7e

More minimal: https://getrector.org/demo/1ec4874f-e24f-63c0-915b-61b98183d7cf

## Expected Behaviour

Unsure, but this is a minized version of a crash case I saw. On non-class code.

```
Fatal error: Uncaught TypeError: PhpParser\Internal\TokenStream::getIndentationBefore(): Return value must be of type int, null returned
```

Basically this is called with a negative number: \PhpParser\Internal\TokenStream::getIndentationBefore(-1)

In the original code the variable given to json_decode() is not unset. But it doesn't matter if you make it a string  `$b =''`. Still crashes.
